checkstyle/suppressions.xml

```diff
@@ -45,6 +45,7 @@
               files="(KafkaClusterTestKit).java"/>
     <suppress checks="JavaNCSS"
               files="RemoteLogManagerTest.java"/>
+    <suppress checks="ClassDataAbstractionCoupling" files="SharePartitionManagerTest"/>
```

Comment:
Can we remove this suppression? We have refactored SharePartition and SharePartitionManager code to have smaller methods so can go away with this suppression.

Reply:
Hi, thanks for the review. Actually I am not able to run any tests locally without the suppression. It results in a checkstyle error without the suppression

Reply:
Yeah, what I am suggesting is that to check if there is a way to write tests better so we do not have to add this suppression.